src/gluonnlp/data/transforms.py

```diff
@@ -693,3 +696,299 @@ def __call__(self, sample):
             Detokenized text
         """
         return self._processor.DecodePieces(sample)
+
+
+class BasicTokenizer():
+    r"""Runs basic tokenization (punctuation splitting, lower casing, etc.).
+
+    Parameters
+    ----------
+    do_lower_case : bool, default True
+        If model is uncased, then do_lower_case should be set to True
```

Comment:
remember that this is a general tokenizer class now and not specific to BERT. the document should clearly define what the tokenizer does, instead of relying on information from model.